src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -293,6 +293,34 @@ void regclass_graph_Model(py::module m) {
             :type name: str
             )");
 
+            model.def("reshape",
+                [](ov::Model& self, const std::vector<std::vector<int64_t>>& input_shapes) {
```

Comment:
@praasz could you please tell us, should this new `reshape` method also support `variables_shapes` argument as the rest of `reshape` methods?

Reply:
It should be optional input, but it maybe required to reshape correctly when variables are inside the model